jmx-scraper/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ dependencies {
   runtimeOnly("io.opentelemetry:opentelemetry-exporter-otlp")
   runtimeOnly("io.opentelemetry:opentelemetry-exporter-logging")
 
-  implementation("io.opentelemetry.instrumentation:opentelemetry-jmx-metrics")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-jmx-metrics:2.10.0-alpha")
```

Comment:
Moving contrib before instrumentation in the release allows to remove this circular dependency if we move the JMX insights code directly into contrib and then reuse it as-is in instrumentation. Do you think this would be something relevant to do ? It also changes a bit the "ownership" of the JMX-related code in the instrumentation by relying on the component owners of contrib instead.

Reply:
I think I'd prefer to go in the other direction and keep / move things that we release as part of the core java agent distribution into the java instrumentation repo

@jackshirazi would this address your concern about release ordering, and we could go back to releasing instrumentation repo first and then contrib repo?